src/components/views/read/cvForm/Portfolio.tsx

```diff
@@ -0,0 +1,293 @@
+
+  const [activeCardIndex, setActiveCardIndex] = useState(0);
+  const handleRemove = (cardId: number): void => {
+    if (cards.length === 1) {
+      return; // Do not allow removing the only card
+    }
+
+    const updatedCards = cards.filter((card) => card.id !== cardId);
+    setCards(updatedCards);
+  };
+  const handleEdit = (cardId: number): void => {
+    // Find the index of the card being edited
+    const cardIndex = cards.findIndex((card) => card.id === cardId);
+
+    // Update the active card index
+    setActiveCardIndex(cardIndex);
+    const updatedShowFields = [...showFields];
+
+    // Update the visibility for the selected card
+    updatedShowFields[cardIndex] = true;
+
+    // Update the state with the new visibility array
+    setShowFields(updatedShowFields);
+  };
+  const handleDone = (cardId: number): void => {
+    const updatedShowFields = [...showFields];
+    const cardIndex = cards.findIndex((card) => card.id === cardId);
+    updatedShowFields[cardIndex] = false; // Set the visibility for the selected card to false
+    setShowFields(updatedShowFields); // Update the state with the new visibility array
+  };
+  const [inputValues, setInputValues] = useState<
+    { cardId: number; values: { [key: string]: string } }[]
+  >([]);
+  const handleChange = (cardId: number, field: string, value: string): void => {
+    setInputValues((prevInputValues) => {
+      const cardIndex = prevInputValues.findIndex(
+        (inputValue) => inputValue.cardId === cardId,
+      );
+
+      if (cardIndex === -1) {
+        // Card not found in the input values array, add a new entry
+        return [...prevInputValues, { cardId, values: { [field]: value } }];
+      }
+
+      // Card found in the input values array, update the existing entry
+      return prevInputValues.map((inputValue) => {
+        if (inputValue.cardId === cardId) {
+          return {
+            ...inputValue,
+            values: { ...inputValue.values, [field]: value },
+          };
+        }
+        return inputValue;
+      });
+    });
+  };
+  const [startDate, setStartDate] = useState<Dayjs | null>(dayjs('2022-04-17'));
+  const [endDate, setEndDate] = useState<Dayjs | null>(dayjs('2022-04-18'));
+  const handleNext = (): void => {
+    const modifiedValues = cards.map((card) => {
+      const cardInputValues = inputValues.find(
+        (inputValue) => inputValue.cardId === card.id,
+      );
+
+      return {
+        'Project Title': cardInputValues?.values.projectTitle || '',
+        'Project Description': cardInputValues?.values.projectDesc || '',
+        'Start Date': cardInputValues?.values.startDate || '',
+        'End Date': cardInputValues?.values.endDate || '',
+        'Project Link': cardInputValues?.values.projectLink || '',
+      };
+    });
+    handleValues('Portfolio', modifiedValues);
+    nextPage();
+    nextStep();
+  };
+  return (
+    <div>
+      <h2>Portfolio</h2>
+      <div>
```

Comment:
You could use a sort of caroussel, or a scrolldown list for the cards and adapt the styling for this. When you add a third card, the layout falls apart.